frameworks/cassandra/tests/test_recovery.py

```diff
@@ -27,6 +27,17 @@ def configure_package(configure_universe):
 
 @pytest.mark.sanity
 @sdk_utils.dcos_1_9_or_higher # dcos task exec not supported < 1.9
+def test_node_replace_replaces_seed_node():
```

Comment:
Now that I think about it, I believe there's another way to determine whether a node has joined a ring successfully or not.  `nodetool gossipinfo` I think.  Created [CASSANDRA-644]( https://jira.mesosphere.com/browse/CASSANDRA-644) to look into it.